lib/ansible_tower_client/base_model.rb

```diff
@@ -82,10 +82,11 @@ def self.create(*args)
     def update_attributes!(attributes)
       @api.patch(url, attributes.to_json)
       attributes.each do |method_name, value|
-        if override_raw_attributes[method_name]
-          send("#{override_raw_attributes[method_name]}=", value)
+        invoke_name = "#{override_raw_attributes[method_name] ? override_raw_attributes[method_name] : method_name}="
```

Comment:
`override_raw_attributes[method_name] ? override_raw_attributes[method_name] : method_name`
can be simplified to:
`override_raw_attributes[method_name] || method_name`